server/sonar-web/src/main/js/components/workspace/views/viewer-view.js

```diff
@@ -29,22 +33,41 @@ export default BaseView.extend({
     this.showViewer();
   },
 
-  showViewer () {
-    const that = this;
-    const viewer = new SourceViewer();
-    const options = this.model.toJSON();
-    viewer.open(this.model.get('uuid'), { workspace: true });
-    viewer.on('loaded', () => {
-      that.model.set({
-        name: viewer.model.get('name'),
-        q: viewer.model.get('q')
-      });
-      if (options.line != null) {
-        viewer.highlightLine(options.line);
-        viewer.scrollToLine(options.line);
+  scrollToLine (line) {
+    const row = this.$el.find(`.source-line[data-line-number="${line}"]`);
+    if (row.length > 0) {
+      const sourceViewer = this.$el.find('.source-viewer');
+      let p = sourceViewer.scrollParent();
+      if (p.is(document) || p.is('body')) {
+        p = $(window);
       }
-    });
-    this.viewerRegion.show(viewer);
+      const pTopOffset = p.offset() != null ? p.offset().top : 0;
+      const pHeight = p.height();
+      const goal = row.offset().top - pHeight / 3 - pTopOffset;
+      /* eslint-disable no-console */
+      console.log(goal, p);
```

Comment:
![CRITICAL](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-critical.png 'Severity: CRITICAL') Remove this logging statement. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=javascript%3AS2228)